proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
Updating quota values happens completely thru mixer. 
Changing the set of quotas that apply will require a proxy config change.

This is a regression in function when we moved away from check based quota.
Now mixer api server will need to call proxy config api server to co ordinate this change.  

A co-ordination of this kind was always in the plan for versioned config rollout, we will need to move it up now.

Is it possible attach the quota aspect back to the `check` api method?